example/src/index.css

```diff
@@ -8,11 +8,14 @@
 }
 
 * {
-    margin: 0;
     color: #e7ece9;
     font-family: ExpensifyMono-Neue;
 }
 
+*:not(div) {
```

Comment:
why do we need this?

Reply:
Hm, I remember I added this line, but don't remember why 🙂 
Let me check it one more time.

Reply:
I need this because I want to normalize default margins of tags like `body`, `h1`, etc. However, I don't want to rewrite custom margins of `divs`.

Reply:
Also, it is only for `example`. So it does not affect on a package itself.

Reply:
Okay, thank you:)